app/src/main/java/ca/antonious/sampleapp/KotlinSampleActivity.kt

```diff
@@ -62,11 +63,24 @@ class KotlinSampleActivity : AppCompatActivity() {
         locale_spinner.setSelection(allLocales.indexOf(Locale.getDefault()))
 
         locale_spinner.onItemSelectedListener = object : AdapterView.OnItemSelectedListener {
-            override fun onNothingSelected(parent: AdapterView<*>?) { }
+            override fun onNothingSelected(parent: AdapterView<*>?) {}
             override fun onItemSelected(parent: AdapterView<*>?, view: View?, position: Int, id: Long) {
                 materialDayPicker.locale = allLocales[position]
             }
         }
+
+        val allWeekdays = MaterialDayPicker.Weekday.values()
+        val defaultFirstDayOfWeek = MaterialDayPicker.Weekday.getFirstDayOfWeekFor(Locale.getDefault())
+
+        first_day_of_week_spinner.adapter = ArrayAdapter(this, R.layout.spinner_text_view, allWeekdays)
+        first_day_of_week_spinner.setSelection(allWeekdays.indexOf(defaultFirstDayOfWeek))
```

Comment:
Same comment here. Let's update the locale spinner listener to update the `firstDayOfWeek` selection.

Reply:
Fixed